crates/biome_analyze/src/rule.rs

```diff
@@ -373,6 +373,10 @@ pub enum RuleDomain {
     Solid,
     /// Next.js framework rules
     Next,
+    /// Vue.js 2 & 3 framework rules
+    Vue,
```

Comment:
Since Biome wants to target a more modern stack, I suggest starting with Vue 3. Plus, Vue 2 has reached EOL. It's the same approach we use for lint rules coming from other sources: let's be patient and see if there is enough request for "old" options. 

https://v2.vuejs.org/eol/